utils.py

```diff
@@ -20,15 +20,27 @@ def validate_timestamp(start, end):
     return True
 
 
+def convert_to_iso(timestamp):
+    """Convert a value to ISO-8601 format."""
+    try:
+        timestamp = float(timestamp)
+        iso = '%Y-%m-%dT%H:%M:%SZ'
+        timestamp = datetime.utcfromtimestamp(timestamp).strftime(iso)
+        return timestamp
+    except ValueError:
+        log.error(f'Error: Timestamp value \'{timestamp}\' is not in '
+                  'ISO-8601 format.')
+    except OverflowError:
+        log.error(f'Error: Timestamp \'{timestamp}\' float value is too '
+                  'large to be used as datetime.')
+
+
 def iso_format_validation(timestamp):
     """Verify if a timestamp is in isoformat. If it's not try to convert it."""
```

Comment:
Update docstring to reflect changes in code.

Reply:
Docstring updated.